cluster/definition.go

```diff
@@ -246,27 +285,51 @@ func (d Definition) MarshalJSON() ([]byte, error) {
 		return nil, errors.Wrap(err, "definition hash")
 	}
 
-	// Marshal json version of lock
-	resp, err := json.Marshal(definitionJSON{
-		Name:                d.Name,
-		UUID:                d.UUID,
-		Version:             d.Version,
-		Timestamp:           d.Timestamp,
-		NumValidators:       d.NumValidators,
-		Threshold:           d.Threshold,
-		FeeRecipientAddress: d.FeeRecipientAddress,
-		WithdrawalAddress:   d.WithdrawalAddress,
-		DKGAlgorithm:        d.DKGAlgorithm,
-		ForkVersion:         d.ForkVersion,
-		Operators:           d.Operators,
-		ConfigHash:          configHash[:],
-		DefinitionHash:      defHash[:],
-	})
-	if err != nil {
-		return nil, errors.Wrap(err, "marshal lock")
-	}
+	if isJSONv1x1(d.Version) {
```

Comment:
switch case would be more cleaner + readable imo

Reply:
fair